features/step_definitions/metering.rb

```diff
@@ -167,32 +163,45 @@
     raise "report '#{report_type}' for project '#{namespace}' not found after #{seconds} seconds"
   end
 end
+
 Given /^I enable route for#{OPT_QUOTED} metering service$/ do | metering_name |
   metering_name ||= metering.name
-  htpasswd = BushSlicer::SSL.sha1_htpasswd(username: user.name, password: user.password)
-  cookie_seed = rand_str(32, :hex)
-  route_yaml = <<BASE_TEMPLATE
-    apiVersion: metering.openshift.io/v1alpha1
-    kind: Metering
-    metadata:
-      name: "#{metering.name}"
-    spec:
-      reporting-operator:
-        spec:
-          route:
-            enabled: true
-          authProxy:
-            enabled: true
-            htpasswdData: |
-              #{htpasswd}
-            cookieSeed: "#{cookie_seed}"
-            subjectAccessReviewEnabled: true
-            delegateURLsEnabled: true
+  # create the route only if one does not exists (currently it's hardcoded
+  # in the chart file charts/reporting-operator/values.yaml)
+  unless route('metering').exists?
```

Comment:
I think that you need to be in the project where this route lives. Will be more reliable to specify project in the `route` call.

Reply:
ok, done.